app/controllers/preauthorize_transactions_controller.rb

```diff
@@ -1,263 +1,382 @@
+# coding: utf-8
```

Comment:
That's a hint to the interpreter to use UTF-8 instead of default US-ASCII. Emacs adds those shebangs automatically.